[2023-07-11T14:52:57.613+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: import_data.create_drivers_table manual__2023-07-11T14:52:55.560114+00:00 [queued]>
[2023-07-11T14:52:57.626+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: import_data.create_drivers_table manual__2023-07-11T14:52:55.560114+00:00 [queued]>
[2023-07-11T14:52:57.628+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-11T14:52:57.651+0000] {taskinstance.py:1327} INFO - Executing <Task(PostgresOperator): create_drivers_table> on 2023-07-11 14:52:55.560114+00:00
[2023-07-11T14:52:57.657+0000] {standard_task_runner.py:57} INFO - Started process 7983 to run task
[2023-07-11T14:52:57.666+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'import_data', 'create_drivers_table', 'manual__2023-07-11T14:52:55.560114+00:00', '--job-id', '225', '--raw', '--subdir', 'DAGS_FOLDER/import_data.py', '--cfg-path', '/tmp/tmpprxij6rp']
[2023-07-11T14:52:57.667+0000] {standard_task_runner.py:85} INFO - Job 225: Subtask create_drivers_table
[2023-07-11T14:52:57.737+0000] {task_command.py:410} INFO - Running <TaskInstance: import_data.create_drivers_table manual__2023-07-11T14:52:55.560114+00:00 [running]> on host 1d6232dcfa4d
[2023-07-11T14:52:57.852+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='import_data' AIRFLOW_CTX_TASK_ID='create_drivers_table' AIRFLOW_CTX_EXECUTION_DATE='2023-07-11T14:52:55.560114+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-11T14:52:55.560114+00:00'
[2023-07-11T14:52:57.857+0000] {sql.py:265} INFO - Executing: DROP TABLE IF EXISTS drivers_data;
CREATE TABLE drivers_data (
    "school_year" TEXT,
    "vendor_name" TEXT,
    "type_of_service" TEXT,
    "active_employees" TEXT,
    "job_type" TEXT,
);
[2023-07-11T14:52:57.869+0000] {base.py:73} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2023-07-11T14:52:58.008+0000] {base.py:73} INFO - Using connection ID 'tutorial_pg_conn' for task execution.
[2023-07-11T14:52:58.014+0000] {sql.py:375} INFO - Running statement: DROP TABLE IF EXISTS drivers_data;
CREATE TABLE drivers_data (
    "school_year" TEXT,
    "vendor_name" TEXT,
    "type_of_service" TEXT,
    "active_employees" TEXT,
    "job_type" TEXT,
);, parameters: None
[2023-07-11T14:52:58.016+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/operators/sql.py", line 277, in execute
    **extra_kwargs,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 349, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 380, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.SyntaxError: syntax error at or near ")"
LINE 8: );
        ^

[2023-07-11T14:52:58.032+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=import_data, task_id=create_drivers_table, execution_date=20230711T145255, start_date=20230711T145257, end_date=20230711T145258
[2023-07-11T14:52:58.050+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 225 for task create_drivers_table (syntax error at or near ")"
LINE 8: );
        ^
; 7983)
[2023-07-11T14:52:58.076+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-11T14:52:58.156+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
